util/cron/nightly_email_file.pm

```diff
@@ -43,7 +43,7 @@ sub nightly_email_file{
     # Nothing sorts the system log for us; do that now, so that 'comm' is
     # given a lexically sorted input as it expects.
     $sortedmysystemlog = $mysystemlog . ".sorted";
-    `LC_ALL=C sort $mysystemlog > $sortedmysystemlog`;
+    `LC_ALL=C sort $mysystemlog | grep -v '^\\s*\$' > $sortedmysystemlog`;
```

Comment:
Note to reviewer: I added this because the mysystemlog now has blank entries for successful commands, where before it had nothing. This filters out empty lines.